src/dotnet-install.ps1

```diff
@@ -805,6 +814,10 @@ function Extract-Dotnet-Package([string]$ZipPath, [string]$OutPath) {
             }
         }
     }
+    catch
+    {
+        Say-Error "Failed to extract package. Exception: $_"
+    }
```

Comment:
Not having this catch block was resulting in the exception bubble up, print the error and terminate the execution.
With this new change, we will print the exception, but continue execution and happily print "Installation succeeded" in the end.

We should probably rethrow.

Reply:
done